engine/share/includes/com04_c.inc

```diff
@@ -47,7 +47,8 @@ C.../COM04/
      .   ISFEM,NLOADP_F, NLOADP_B, NUMNOR, NINTER25,
      .   NINTER25E,NUMPLY,NUMSTACK,NS10E,NINTERFRIC,
      .   NUMSKIN,NBCSCYC,NRBFAIL,NLOADP_HYD,NINTLOADP,
-     .   NINTLOADP21,NLOADP_HYD_INTER,NTSHEG,NRBODY0
+     .   NINTLOADP21,NLOADP_HYD_INTER,NTSHEG,NRBODY0,
```

Comment:
Addition of variable in common blocks are note authorized anymore. Could you move this variable into a module? (maybe drape_mod.F if it makes sense)